EIPS/eip-6268.md

```diff
@@ -0,0 +1,94 @@
+---
+eip: 6268
+title: Untransferability Indicator for EIP-1155
+description: An extension of EIP-1155 for indicating the transferability of the token.
+author: Yuki Aoki (@yuki-js)
+discussions-to: https://ethereum-magicians.org/t/sbt-implemented-in-erc1155/12182
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-01-06
+requires: 1155
+---
+
+## Abstract
+
+The following standard is an extension of [EIP-1155](./eip-1155.md). It introduces the interface for indicating whether the token is transferable or not, without regard to non-fungibility, using the feature detection functionality of [EIP-165](./eip-165.md).
+
+## Motivation
+
+We propose the introduction of the UNTransferability Indicator, a universal indicator that demonstrates untransferability without regard to non-fungibility. This will enable the use of Soulbound Tokens (SBT), which are untransferable and fungible/non-fungible entities, to associate items with an account, user-related information, memories, and event attendance records, in a universal manner. The [EIP-5192](./eip-5192.md) specification was invented for this purpose, but SBT in [EIP-5192](./eip-5192.md) is non-fungible and has a tokenId, allowing them to be distinguished from each other using the tokenId. However, for example, in the case of event attendance records, it is not necessary to distinguish between those who attended the same event using the tokenId, and all participants should have the same indistinguishable entity. Rather, the existence of the tokenId creates discriminability.
+
+Additionally, there is currently no mechanism to indicate untransferability in [EIP-20](./eip-20.md) or [EIP-1155](./eip-1155.md). Therefore, a universal specification with the same functionality as [EIP-5192](./eip-5192.md), which can be applied to more than just [EIP-721](./eip-721.md), is required.
+
+Therefore, we propose the introduction of a universal indicator that demonstrates untransferability without regard to non-fungibility, inspired by [EIP-5192](./eip-5192.md). This will make it impossible to identify individuals when using SBT to associate information, protecting the privacy of personal information. It will also make it possible to choose a method other than [EIP-721](./eip-721.md) when implementing SBT. Then tokens with UNTransferability Indicator become something like account-bound attributes rather than assets.
```

Comment:
```suggestion
Soulbound Tokens (SBT) are non-transferable tokens. While [EIP-5192](./eip-5192.md) standardizes non-fungible SBTs, a standard for Soulbound semi-fungible or fungible tokens does not yet exist. The introduction of a standard non-transferability indicator that is agnostic to fungibility promotes the usage of Souldbound semi-fungible or fungible tokens.
```
This can be shortened to one paragraph.